bin/phpcs.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+REM If you have problems, run this in its directory, else place phpstancs/bin at the head of your PATH
+
+phpcs %1 %2 %3 %4 %5 %6 %7 %8 %9
```

Comment:
- missing php executable
- try %*  to pass all arguments
- call phpsc with full path eg. using %~dp0

Reply:
this working for me
```suggestion
@echo off

if "%PHP_PEAR_PHP_BIN%" neq "" (
    set PHPBIN=%PHP_PEAR_PHP_BIN%
) else set PHPBIN=php

"%PHPBIN%" "%~dp0\phpcs" %*
```